tests/results/result_actions/ApplyPatchActionTest.py

```diff
@@ -115,7 +115,8 @@ def test_apply_rename(self):
                       file_diff_dict)
             self.assertFalse(isfile(f_a+'.renamed.orig'))
 
-            file_dict = {f_a+'.renamed': open(f_a+'.renamed').readlines()}
+            with open(f_a+'.renamed') as fh:
```

Comment:
this one looks different from the other ones ... it should be a separate PR or separate commit in this PR.